edition = "2023";

package cloud_databases_nl2code_eval_dbcodegen_proto;

option java_multiple_files = true;

message EvalResponse {
  string response = 1;
  string session_id = 2;
}

